views/pages/login.ejs

```diff
@@ -2,5 +2,6 @@
 <%- include('../partials/header')-%>
 <!-- Page Content Starts -->
     <h1>Login Page</h1>
+<%- include('../partials/footer')-%>
 </body>
```

Comment:
@ZM202 Lets remove the closing body and the closing html tag and add them into the footer.ejs file.

Reply:
All set